usr/share/rear/backup/BORG/default/50_make_backup.sh

```diff
@@ -0,0 +1,28 @@
+# This file is part of Relax-and-Recover, licensed under the GNU General
+# Public License. Refer to the included COPYING for full text of license.
+#
+# 20_start_backup.sh
+
+include_list=()
+
+# Check if backup-include.txt (created by 40_create_include_exclude_files.sh),
+# really exists
+if [ ! -r $TMP_DIR/backup-include.txt ]; then
+    Error "Cant find include list"
+fi
+
+# Create Borg friendly include list
+for i in $(cat $TMP_DIR/backup-include.txt); do
+    include_list+=("$i ")
+done
+
+Log "Creating archive ${BORG_ARCHIVE_PREFIX}_$SUFFIX \
+in repository $BORG_REPO on host $BORG_HOST"
+
+# Start actual Borg backup
+borg create --one-file-system --verbose --stats \
```

Comment:
Rear uses '--one-file-system' intentionally for the 'tar' backup
so that other backup methods should behave by default same
but it could also depend on the other backup method what
its particular default behaviour is - i.e. different backup methods
could behave differently in rear.